moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -802,16 +802,32 @@ bool ServoCalcs::applyJointUpdate(const Eigen::ArrayXd& delta_theta, sensor_msgs
 
   for (std::size_t i = 0; i < joint_state.position.size(); ++i)
   {
-    // Increment joint
-    joint_state.position[i] += delta_theta[i];
+    // Increment joints
+
+    // If no joint drift thresholding, simply increment all joints
+    if (!parameters_->apply_anti_drift)
+    {
+      joint_state.position.at(i) += delta_theta[i];
+    }
+    // Else, increment if Servo is actuating the joint, i.e. delta_theta[i] != 0
+    else if (delta_theta[i] != 0)
+    {
+      joint_state.position.at(i) += delta_theta[i];
+    }
+    // Else, keep the joint's original position from original_joint_state_
+    // This is what mitigates drift.
+    else
+    {
+      joint_state.position.at(i) != original_joint_state_.position.at(i);
```

Comment:
This should just be an `=` operator, yeah?

Reply:
yep, good catch. Obviously I didn't test this with anti-drift enabled.

Reply:
i've tested now with the Panda in sim and it seems fine